lib/fastlane/plugin/maestro_orchestration/actions/maestro_orchestration_android_action.rb

```diff
@@ -18,6 +18,12 @@ def self.run(params)
         end
 
         UI.message("--------------\n\nSDK DIR: #{params[:sdk_dir]}\n\n--------------")
+        if params[:clear_maestro_logs]
+          UI.message("Clearing previous Maestro logs...")
+          sh("rm -rf ~/.maestro/tests/*")
```

Comment:
Can we do something in Ruby? I don't trust a rm -rf ~/ as folks could have aliases set, etc. Its too dangerous.

```ruby
  Dir.glob(File.expand_path("~/.maestro/tests/*")).each do |file|
    if File.directory?(file)
      Dir.rmdir(file) rescue nil
    else
      File.delete(file) rescue nil
    end
  end
  ```
  
  might work

Reply:
that could be a helper ^